tests/typescript/conformance/es6/Symbols/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -19,7 +19,7 @@ bar(i);~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 //@target: ES6
 class C {}
 interface I {
-  [Symbol.iterator]?: { x };
+  [Symbol.iterator]: { x };
```

Comment:
It's a regression from https://github.com/eslint/typescript-eslint-parser/pull/292#issuecomment-304528291